tests/integration/assets/dids/issuer/did.json

```diff
@@ -0,0 +1,32 @@
+{
+  "@context": [
+    "https://www.w3.org/ns/did/v1",
+    "https://w3id.org/security/suites/jws-2020/v1"
+  ],
+  "id": "did:web:raw.githubusercontent.com:digicatapult:veritable-cloudagent:main:tests:integration:assets:dids:issuer",
+  "verificationMethod": [
+    {
+      "id": "did:web:raw.githubusercontent.com:digicatapult:veritable-cloudagent:main:tests:integration:assets:dids:issuer#owner",
+      "type": "JsonWebKey2020",
+      "controller": "did:web:raw.githubusercontent.com:digicatapult:veritable-cloudagent:main:tests:integration:assets:dids:issuer",
+      "publicKeyJwk": {
+        "kty": "OKP",
+        "crv": "Ed25519",
+        "x": "Kf6_nD9O5K4rgWoyLICXmAGyd_shjBCai5hUcyi_Smo"
+      }
+    }
+  ],
+  "authentication": ["did:web:raw.githubusercontent.com:digicatapult:veritable-cloudagent:main:tests:integration:assets:dids:issuer#owner"],
+  "assertionMethod": ["did:web:raw.githubusercontent.com:digicatapult:veritable-cloudagent:main:tests:integration:assets:dids:issuer#owner"],
+  "service": [{
+    "id": "did:web:raw.githubusercontent.com:digicatapult:veritable-cloudagent:main:tests:integration:assets:dids:issuer#did-communication",
+    "type": "did-communication",
+    "priority" : 0,
+    "recipientKeys" : [ "did:web:raw.githubusercontent.com:digicatapult:veritable-cloudagent:main:tests:integration:assets:dids:issuer#owner" ],
+    "routingKeys" : [],
+    "accept": [
+      "didcomm/aip1"
+    ],
+    "serviceEndpoint": "http://issuer:5002"
```

Comment:
should this be charlie (based on docker composes in this repo)?

Reply:
yes! thank you, good catch - getting mixed up with nice deployment